src/SimulatedEffect/Task.elm

```diff
@@ -98,6 +98,16 @@ fail =
     SimulatedEffect.Fail
 
 
+{-| Start with a list of tasks, and turn them into a single task that returns a
+list. The tasks will be run in order one-by-one and if any task fails the whole
+sequence fails.
+sequence [ succeed 1, succeed 2 ] == succeed [ 1, 2 ]
+-}
```

Comment:
I've been limiting the docs here to just the first sentence of the real module's doc to avoid having a ton of copied text.

```suggestion
{-| Start with a list of tasks, and turn them into a single task that returns a
list.
-}
```